src/lib/dom-api.html

```diff
@@ -593,6 +593,55 @@
 
     };
 
+    Object.defineProperties(DomApi.prototype, {
+      activeElement: {
+        get: function() {
+          var ownerDocument = this.node.ownerDocument;
+          var trueDocument = ownerDocument || this.node;
+
+          // Native, non-document ShadowRoot?
+          if (this.node.host && ownerDocument && !window.ShadowDOMPolyfill && !this.node._isShadyRoot) {
+            return this.node.activeElement;
+          }
+
+          // Not ShadowRoot-like?
+          if (!this.node.host && ownerDocument) {
+            return this.node.activeElement;
+          }
+
+          var active = trueDocument.activeElement;
+          if (!active) return null;
+
+          if (window.ShadowDOMPolyfill) {
```

Comment:
[Doesn't look like it.](https://github.com/webcomponents/webcomponentsjs/search?q=activeElement)